modules/linear_boltzmann_solvers/lbs_problem/lbs_problem.cc

```diff
@@ -52,15 +52,18 @@ LBSProblem::GetInputParameters()
 
   params.AddRequiredParameter<std::shared_ptr<MeshContinuum>>("mesh", "Mesh");
 
-  params.AddRequiredParameter<size_t>("num_groups", "The total number of groups within the solver");
+  params.AddRequiredParameter<int>("num_groups", "The total number of groups within the solver");
```

Comment:
This should have stayed as `size_t`, right? Or do we assume negative number of groups?